requirements.txt

```diff
@@ -68,7 +68,6 @@ scikit-learn==0.21.3
 six==1.13.0
 smmap2==2.0.5
 sqlparse==0.2.4
```

Comment:
Here, removing `stringsifter` won't be enough. We should also remove the entries for all it's sub-dependencies like numpy, etc. Should use something like pipdeptree (https://pypi.org/project/pipdeptree/) to inspect.